tools/syz-imagegen/imagegen.go

```diff
@@ -764,26 +782,29 @@ func runCmd(cmd string, args ...string) ([]byte, error) {
 	return osutil.RunCmd(10*time.Minute, "", cmd, args...)
 }
 
-func writeImage(image *Image, data []byte) ([]byte, error) {
+func writeImage(image *Image, length int, segments []Segment, chdir bool) []byte {
```

Comment:
Do we need to write 2 versions for each image? It does not seem to give any new coverage as is and the fuzzer can flip the chdip argument in programs in the corpus.
There are lots of these seeds already and they are huge.
Maybe let's write only with chdir=true. syzbot corpuses already contain lots of existing seeds and mutated seeds, all of these will be transformed to chdir=false. So it will effectively achieve roughly the same effect.